projects/vanilla/src/app/search/search.component.html

```diff
@@ -152,8 +144,16 @@
                     <sq-tabs [results]="results" [showCounts]="!config.hideCounts"></sq-tabs>
                 </ng-template>
 
-                <ng-template uib-template="my-search" display="My search" let-config>
-                    <sq-facet-mysearch [results]="results" [allowDeletion]="config.allowDeletion"></sq-facet-mysearch>
+                <ng-template uib-template="filters-view" display="Filters view" let-config>
+                  <div class="d-flex small sq-toolbar">
```

Comment:
I think that you can remove this div (the "small" class could go in the container configuration)

Reply:
done